scripts/GameManager.gd

```diff
@@ -49,3 +49,25 @@ func _on_level_up_continue_game():
 	$now_enemies.delete_enemies()
 	pass # Replace with function body.
 
+
+# When the player is dead
+func _on_player_dead():
+	print("Player dead")
+	get_tree().paused = true
+	$GameUI.timer_stop()
+	$Dead.init()
+	
+func _on_dead_restart_game():
```

Comment:
Hi, could u please take a look at this function to restart the round? I totally have no idea why my restart button cannot trigger this function.
Many thanks
@Si1w